app/services/hmpps_api/prison_api/agencies_api.rb

```diff
@@ -0,0 +1,21 @@
+module HmppsApi
+  module PrisonApi
+    class AgenciesApi
+      extend PrisonApiClient
+      HOSPITAL_AGENCY_TYPE = 'HSHOSP'.freeze
+      def self.agencies_by_type(type)
+        route = "/agencies/type/#{type}"
```

Comment:
I'd suggest pulling this out to a global constant which can be interpolated in to the string as it's used in two places.

Reply:
Just to clarify, are you suggesting HOSPITAL_AGENCY_TYPE should be declared as a rails wide constant? Or should it be under a module for more context?

Reply:
Yes, just inside this class.

Reply:
Apologies, I'm still not 100% on what you are suggesting, probably down to me being dense in this instance. I could specify a true global using $HOSPITAL_AGENCY_TYPE, or even add the value to the config options. Alternatively I could add the Constant at a higher level within the module stack, such as HmppsApi::HOSPITAL_AGENCY_TYPE

If you have an example of how you would reference the constant from within the app, I should be fine to implement it.

Reply:
Oh sorry, I'm referring to the `route = "/agencies/type/#{type}"` line rather than `HOSPITAL_AGENCY_TYPE`.

```ruby
BASE_PATH = "/agencies/type".freeze

... # and then it's used

route = URI::join(BASE_PATH, type)
```

Reply:
Ah understood now :) I'll make the changes and push up. Thanks for the advice.